tests/integration/api/v1/curated_recommendations/test_curated_recommendations.py

```diff
@@ -1,46 +1,114 @@
+    mock_http_client = AsyncMock(spec=AsyncClient)
+
+    # Mock the POST request response with the loaded json mock data
+    mock_http_client.post.return_value = Response(
+        status_code=200,
+        json=scheduled_surface_data,
+        request=Request(
+            method="POST",
+            url=CorpusApiGraphConfig.CORPUS_API_PROD_ENDPOINT,
+            headers=CorpusApiGraphConfig.HEADERS,
+            json={"locale": "en-US"},
+        ),
+    )
+
+    return mock_http_client
+
+
+@pytest.fixture(name="corpus_backend")
+def fixture_mock_corpus_backend(corpus_http_client: AsyncMock) -> CorpusApiBackend:
+    """Mock corpus api backend."""
+    # Initialize the backend with the mock HTTP client
+    return CorpusApiBackend(http_client=corpus_http_client)
+
+
+@pytest.fixture
+def provider(corpus_backend):
+    """Mock corpus provider."""
+    return CuratedRecommendationsProvider(corpus_backend=corpus_backend)
+
+
+@pytest.fixture(autouse=True)
+def setup_providers(provider):
+    """Set up the corpus api provider"""
+    corpus_api_provider["corpus_provider"] = provider
+
+
 @freezegun.freeze_time("2012-01-14 03:21:34", tz_offset=0)
-def test_curated_recommendations_locale(client: TestClient) -> None:
-    """Test if curated-recommendations endpoint returns result."""
+@pytest.mark.asyncio
+async def test_curated_recommendations_locale(client: TestClient):
+    """Test the curated recommendations endpoint response is as expected."""
+    # expected recommendation with topic = None
     expected_recommendation = CuratedRecommendation(
         scheduledCorpusItemId="50f86ebe-3f25-41d8-bd84-53ead7bdc76e",
         url=HttpUrl("https://www.themarginalian.org/2024/05/28/passenger-pigeon/"),
         title="Thunder, Bells, and Silence: the Eclipse That Went Extinct",
-        excerpt="What was it like for Martha, the endling of her species, to die alone at "
-        "the Cincinnati Zoo that late-summer day in 1914, all the other "
-        "passenger pigeons gone from the face of the Earth, having onc…",
-        topic=Topic.EDUCATION,
+        excerpt="Juneteenth isn’t the “other” Independence Day, it is THE Independence Day.",
+        topic=None,
         publisher="The Marginalian",
         imageUrl=HttpUrl(
-            "https://www.themarginalian.org/wp-content/uploads/2024/05"
-            "/PassengerPigeon_Audubon_TheMarginalian.jpg?fit=1200%2C630&ssl=1"
+            "https://s3.us-east-1.amazonaws.com/pocket-curatedcorpusapi-prod-images/87fd6901-5bf5-4b12-8bde-24b86be79003.jpeg"  # noqa
         ),
-        receivedRank=0,
+        receivedRank=1,
     )
-
+    # Mock the endpoint
     response = client.post("/api/v1/curated-recommendations", json={"locale": "en-US"})
+    data = response.json()
+
+    # Check if the mock response is valid
     assert response.status_code == 200
 
-    result = response.json()
-    assert result["recommendedAt"] == 1326511294000  # 2012-01-14 03:21:34 UTC
+    corpus_items = data["data"]
```

Comment:
At this point, I would call them 'curated recommendations' for consistency.
```suggestion
    curated_recommendations = data["data"]
```